app/src/pages/CollectionPage/CollectionPage.tsx

```diff
@@ -107,6 +114,19 @@ export const CollectionPage = (): React.ReactElement => {
                   <MetricView name={'24h volume'} value={350} />
                 </Stack>
               </Stack>
+              <Stack direction={Direction.Vertical} childAlignment={Alignment.Start} shouldAddGutters={true} paddingVertical={PaddingSize.Wide2}>
+                <Text variant='header3-singleLine'>Your Holdings (2)</Text>
```

Comment:
why is singleLine here?? also you've hardcoded the number of holdings now and its still wrong